Symfony/CS/Tests/Fixer/Symfony/HeredocToNowdocFixerTest.php

```diff
@@ -104,6 +104,36 @@ public function provideTestFixCases()
 
 EOF
             ),
+            array(
+                <<<'EOF'
+<?php
+$html = <<<'HTML'
+a
+HTML;
+
+EOF
+            ,
+                <<<'EOF'
+<?php
+$html = <<<   HTML
+a
+HTML;
+
+EOF
+            ),
+            array(<<<'EOF'
+<?php $a = <<<'TEST'
+Foo
+TEST;
+
+EOF
+            , <<<'EOF'
+<?php $a = <<<           "TEST"
```

Comment:
hmm, this is invalid. why isn't it failing in the travis build with enabled linting?